src/Resources/config/loggable.xml

```diff
@@ -16,13 +16,16 @@
             <call method="setAnnotationReader">
                 <argument type="service" id=".stof_doctrine_extensions.reader" on-invalid="ignore" />
             </call>
+            <call method="setActorProvider">
+                <argument type="service" id="stof_doctrine_extensions.tool.actor_provider" />
+            </call>
         </service>
 
+        <!-- deprecated in favor of the actor provider -->
         <service id="stof_doctrine_extensions.event_listener.logger" class="%stof_doctrine_extensions.event_listener.logger.class%">
```

Comment:
Please add the `<deprecated>` tag to properly mark the listener service as deprecated, and so the `@deprecated` annotation on the listener class (and we can then remove the comment).

Reply:
Updated